packages/flame/lib/src/text/formatter_text_renderer.dart

```diff
@@ -0,0 +1,35 @@
+import 'dart:ui';
+
+import 'package:flame/src/anchor.dart';
+import 'package:flame/src/text/formatters/text_formatter.dart';
+import 'package:flame/text.dart';
+import 'package:vector_math/vector_math_64.dart';
+
+/// Helper class that implements a [TextRenderer] using a [TextFormatter].
+class FormatterTextRenderer<T extends TextFormatter> extends TextRenderer {
```

Comment:
`Formatted` maybe?

Reply:
The etymology here is that this is a `TextRenderer` based on a `TextFormatter`. So, the name is "TextFormatter TextRenderer", where the first "Text" is elided for brevity.

Note also that this is a mostly internal class -- the only situation where it could be useful to a user is if they tried to implement their own `TextRenderer`, which is rather unlikely to happen.